__tests__/integrations.test.js

```diff
@@ -30,6 +30,22 @@ describe('404: Incorrect URL', () => {
   });
 });
 
+// ENDPOINTS DESCRIPTION
+
+describe('ENDPOINTS', () => {
+  test('GET 200: Endpoint returns the endpoints description file detailing all endpoints.', () => {
+    return request(app)
+      .get('/api')
+      .expect(200)
+      .then(({ body }) => {
+        fs.readFile(`${__dirname}/../endpoints.json`, 'utf-8').then((data) => {
```

Comment:
this is synchronous data so you can use either regular fs, doesn't need to be promises, or you can just require in the endpoints.json file